fake-cucumber/javascript/src/FakeStepDefinitions.ts

```diff
@@ -0,0 +1,103 @@
+import { messages } from 'cucumber-messages'
+import {
+  ParameterTypeRegistry
+} from 'cucumber-expressions'
+import uuidv4 from 'uuid/v4'
+
+import Argument from 'cucumber-expressions/dist/src/Argument'
+import CucumberExpression from 'cucumber-expressions/dist/src/CucumberExpression'
+
+class FakeStepDefinition {
+  public id: string
```

Comment:
Try to make fields `private readonly` as much as possible to prevent exposing more internals than necessary